configs/keep-ecdsa/config.local.2.toml

```diff
@@ -1,18 +1,19 @@
 [ethereum]
-    URL = "ws://127.0.0.1:8546"
+  URL = "ws://127.0.0.1:8546"
 
 [ethereum.account]
-    KeyFile = "WORKDIR/ethereum/data/keystore/UTC--2019-08-01T13-19-44.188278000Z--b7021acafecf289fa345ecc95ecab18077530841"
+  KeyFile = "WORKDIR/ethereum/data/keystore/UTC--2019-08-01T13-19-44.188278000Z--b7021acafecf289fa345ecc95ecab18077530841"
 
 [ethereum.ContractAddresses]
   BondedECDSAKeepFactory = "0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+  TBTCSystem = "0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
 
-[SanctionedApplications]
-    Addresses = ["0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"]
+[TSS]
+  PreParamsTargetPoolSize = 2
```

Comment:
@nkuba @lukasz-zimnoch  going to merge this but want to make sure there isn't an unexpected side effect from reducing this for any of the tests y'all use this setup for.